### PR TITLE
Support stdout and stderr streaming in flutter embedder.

### PR DESCRIPTION
**Pull request by turnidge**

### CODE REVIEW

File: sky/engine/bindings/dart_runtime_hooks.cc [hunk 1]
```diff
@@ -37,6 +39,14 @@ extern void syslog(int, const char *, ...);
 }
 #endif
 
+namespace dart {
+namespace bin {
+
+extern bool capture_stdout;
```
Comment (abarth) [inline sky/engine/bindings/dart_runtime_hooks.cc]: We should create a header file that declares these symbols so we can see the dependencies in the codebase by looking at the include rules.

File: sky/engine/core/script/dart_init.cc [hunk 2]
```diff
@@ -44,6 +44,16 @@ extern const uint8_t* observatory_assets_archive;
 }  // namespace observatory
 }  // namespace dart
 
+namespace dart {
+namespace bin {
+
+extern bool capture_stdio;
+extern bool capture_stdout;
+extern bool capture_stderr;
```
Comment (abarth) [inline sky/engine/core/script/dart_init.cc]: ditto

**Comment by abarth** [comment]
Otherwise looks reasonable.

File: sky/engine/bindings/dart_runtime_hooks.cc (hunk 1 above)
Comment (turnidge) [inline sky/engine/bindings/dart_runtime_hooks.cc]: Will do.  Just committed the Dart half of this in 1d2d1ce0fcd83d65c68045222157a0fd5b187781.

**Pull request closed without merging**